src/canvas/Canvas.ts

```diff
@@ -1055,7 +1055,7 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
       );
       if (target === this._activeObject && (corner || !grouped)) {
         this._setupCurrentTransform(e, target, alreadySelected);
-        const control = target.controls[corner],
+        const control = corner ? corner.control : undefined,
```

Comment:
I prefer `corner` being renamed.

Reply:
corner is an internal variable right? if yes it can be renamed of course.

Reply:
Done, although `handle` sounds weird and never used before